src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -248,7 +248,7 @@ public function getBundle($name, $first = true)
      *
      * The resource name must follow the following pattern:
      *
-     *     @<BundleName>/path/to/a/file.something
+     *     "@BundleName/path/to/a/file.something"
      *
      * where BundleName is the name of the bundle
      * and the remaining part is the relative path in the bundle.
```

Comment:
The Fabbot says I should use `{@inheritDoc}` with a lower-cased `d`. Which one is correct now?